### ISSUE TITLE
Inspirational Quotes Feature 

### ISSUE THREAD

**Issue by meghanakn473** [TenzDelek/DearDiary#49]
I want to implement a feature that suggests quotes or prompts to inspire users when they're unsure what to write about.
how i can i implement it

**Comment by meghanakn473** [comment]
@TenzDelek please assign me this issue i want to work on it

**Comment by TenzDelek** [comment]
Can you share what and how you will be implementing it. A board overview, like are you going to use ai to suggest the thing. It would be helpful

**Comment by meghanakn473** [comment]
> Can you share what and how you will be implementing it. A board overview, like are you going to use ai to suggest the thing. It would be helpful

i wanted to implement it from Quotes REST API: [Quotes API](https://quotes.rest/)
what r ur suggestions how can i work on ? @TenzDelek

**Comment by TenzDelek** [comment]
@meghanakn473  looks great , go ahead

**Comment by meghanakn473** [comment]
hey @TenzDelek i need ur help

**Comment by TenzDelek** [comment]
did you run
> git pull

 cc @meghanakn473

**Comment by meghanakn473** [comment]
> did you run
> 
> > git pull
> 
> cc @meghanakn473

yes wen i run npm run dev this error is coming, i provided correct publishable key from clerk but still it shows this

**Comment by TenzDelek** [comment]
Do a npm install

**Comment by TenzDelek** [comment]
Also share your vscode screenshot

**Comment by meghanakn473** [comment]
> Do a npm install

![image](https://github.com/user-attachments/assets/07659113-fcc0-4263-bd8f-7a813bfbc383)

**Comment by meghanakn473** [comment]
@TenzDelek hey did u check  ??

**Comment by TenzDelek** [comment]
i can see pages folder which shouldnt be there, i believe i told you once that this is a next repo. kindly check with the docs once. and regarding the issue, can you share the package.json screenshot or the file here

**Comment by meghanakn473** [comment]
> i can see pages folder which shouldnt be there, i believe i told you once that this is a next repo. kindly check with the docs once. and regarding the issue, can you share the package.json screenshot or the file here

package.json 

{
  "name": "gssocextended",
  "version": "0.1.0",
  "private": true,
  "scripts": {
    "dev": "next dev",
    "build": "next build",
    "start": "next start",
    "lint": "next lint",
    "postinstall": "prisma generate",
    "prepare": "husky install"
  },
  "dependencies": {
    "@clerk/clerk-react": "^5.11.0",
    "@clerk/clerk-sdk-node": "^5.0.48",
    "@clerk/nextjs": "^5.7.1",
    "@prisma/client": "^5.20.0",
    "@radix-ui/react-slot": "^1.1.0",
    "class-variance-authority": "^0.7.0",
    "framer-motion": "^11.11.1",
    "lenis": "^1.1.13",
    "lucide-react": "^0.447.0",
    "next": "14.2.13",
    "next-auth": "^4.24.8",
    "react": "^18",
    "react-dom": "^18",
    "tailwind-merge": "^2.5.3",
    "tailwindcss-animate": "^1.0.7"
  },
  "devDependencies": {
    "@types/node": "22.7.4",
    "@types/react": "18.3.11",
    "eslint": "^8",
    "eslint-config-next": "14.2.13",
    "husky": "^9.1.6",
    "lint-staged": "^15.2.10",
    "postcss": "^8",
    "prisma": "^5.20.0",
    "tailwindcss": "^3.4.1"
  },
  "lint-staged": {
    "*.{js,jsx,ts,tsx}": [
      "eslint --fix",
      "prettier --write"
    ],
    "*.{json,md}": [
      "prettier --write"
    ]
  }
}

**Comment by meghanakn473** [comment]
> i can see pages folder which shouldnt be there, i believe i told you once that this is a next repo. kindly check with the docs once. and regarding the issue, can you share the package.json screenshot or the file here

i get it, whr shd the content present inside the pages folder be placed? @TenzDelek

**Comment by TenzDelek** [comment]
your package.json looks good, and for the the pages part, nextjs follows folder based routing so in app dir, create a inspiration folder and inside that put page.jsx you can paste that code in it.

**Comment by meghanakn473** [comment]
> your package.json looks good, and for the the pages part, nextjs follows folder based routing so in app dir, create a inspiration folder and inside that put page.jsx you can paste that code in it.

in page folder i hv 2 files App.js and quote.js....so both these files i shd put under inspiration folder in page.jsx or shd i create like app.jsx and quote.jsx inside inspiration folder??

**Comment by meghanakn473** [comment]
> > your package.json looks good, and for the the pages part, nextjs follows folder based routing so in app dir, create a inspiration folder and inside that put page.jsx you can paste that code in it.
> 
> @TenzDelek in page folder i hv 2 files App.js and quote.js....so both these files i shd put under inspiration folder in page.jsx or shd i create like app.jsx and quote.jsx inside inspiration folder??

@TenzDelek can u please guide me wt to do so that i can finish with it

**Comment by meghanakn473** [comment]
?? @TenzDelek

**Comment by TenzDelek** [comment]
merge it together, i believe you are importing quote.js to app.js

**Comment by meghanakn473** [comment]
> merge it together, i believe you are importing quote.js to app.js 

I did the same, the page opens 
But under daily quotes, random inspiration quotes r not being displayed

**Comment by TenzDelek** [comment]
> > merge it together, i believe you are importing quote.js to app.js 
> 
> I did the same, the page opens 
> But under daily quotes, random inspiration quotes r not being displayed
> 
> @TenzDelek how do I correct it

A screen recording would be helpful

**Comment by meghanakn473** [comment]
> > > merge it together, i believe you are importing quote.js to app.js 
> > 
> > I did the same, the page opens 
> > But under daily quotes, random inspiration quotes r not being displayed
> > 
> > @TenzDelek how do I correct it
> 
> A screen recording would be helpful 

https://github.com/user-attachments/assets/ced4c0e6-9222-4d86-9fae-92094536d07e

**Comment by meghanakn473** [comment]
// components/QuoteSuggestions.js

import React from 'react';
import quotes from '../data/quotes';
import styles from '../styles/QuoteSuggestions.module.css';

const QuoteSuggestions = () => {
    // Function to randomly select a quote
    const getRandomQuote = () => {
        const randomIndex = Math.floor(Math.random() * quotes.length);
        return quotes[randomIndex].text;
    };

    return (
        <div className="quote-suggestion">
            <h3>Need Inspiration?</h3>
            <p>{getRandomQuote()}</p>
            <button onClick={() => window.location.reload()}>Get Another Quote</button>
        </div>
    );
};

export default QuoteSuggestions;

This is my QuoteSuggestions.js file

**Comment by meghanakn473** [comment]
@TenzDelek ??

**Comment by Aditya-PS-05** [comment]
Can you assign me this issue?
